src/opera/commands/deploy.py

```diff
@@ -19,12 +20,16 @@ def add_parser(subparsers):
         help=".opera storage folder location"
     )
     parser.add_argument(
-        "--inputs", "-i", type=argparse.FileType("r"),
+        "--inputs", "-i", type=str,
         help="YAML file with inputs",
     )
+    parser.add_argument(
+        "--template", "-t", type=str,
+        help="TOSCA YAML service template file",
+    )
```

Comment:
Ok, so now when the CSAR is being initialized I look for the template in `Entry-Definitions` and use it later when deploy is called.

Reply:
But instead of the `csar` positional argument I used `template` to mark that deploy is used with TOSCA service templates and not directly with CSAR files.

Reply:
This `template` positional argument is now optional in case that user wants to do `opera deploy template.yaml` as before but I can remove this argument if you think that it is necessary just to kep the inputs argument and to move everything else to init.

Reply:
I'm good with being able to provide path to a csar or, like in the past, a path to a yaml.

Reply:
Ok great :+1: